tutorials/experiments/llama-index/answer_and_context_relevancy_llamatrace.ipynb

```diff
@@ -0,0 +1,3268 @@
+{
```

Comment:
<p>The arize-phoenix version is gonna have to match the version that is of the llamatrace or hosted phoenix long term. Maybe check the deployed version before the workshop but it is 4.20 at the current moment I believe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>